docs/hazmat/primitives/asymmetric/rsa.rst

```diff
@@ -259,6 +259,11 @@ Valid paddings for encryption are
 is the recommended choice for any new protocols or applications, ``PKCS1v15``
 should only be used to support legacy protocols.
 
+.. note::
+
+    An RSA key can only encrypt data smaller than itself, using this with a message that is too big will raise a ``ValueError``
```

Comment:
Instead of this note at the top, let's include this in the docs for `ValueError` in teh `encrypt` section.

Reply:
Good call, I put it there mostly because it's the one most targeted to newbies, I moved the note under the ValueError section